invenio/b2share/modules/b2deposit/b2share_model/model.py

```diff
@@ -95,13 +96,11 @@ def __init__(self):
                            'its relevance quickly.'
         }
         self.field_args['publisher'] = {
-            'hidden': True,
-            'value': self.publisher_default
-            # 'description':
-            # 'Here should be stored the site that will host the BE2Share ' +\
-            # 'container, so that in case of access problems, people can ' +\
-            # 'be contacted. This element can be created automatically ' +\
-            # 'dependent on the centre.'
+            'value': self.publisher_default,
+            'description': 'Here should be stored the site that will host the B2SHARE '
```

Comment:
Agree, but what "should" the field represent? The name, description, common use and our use are not consistent. We assign it automatically a "service instance url" but it's unclear to me why so, and e.g. an article could already have been published in a magazine to which the uploader wants it to refer.

@cjhak can you maybe give some input on this? Specifically use and description.